src/js/multiDomainFirstParties.js

```diff
@@ -306,17 +306,6 @@ var multiDomainFirstPartiesArray = [
   ],
   ["gotomeeting.com", "citrixonline.com"],
   ["guardian.co.uk", "guim.co.uk", "guardianapps.co.uk", "theguardian.com", "gu-web.net"],
-  [
```

Comment:
Looks like this file got updated by accident.

Reply:
Fixed.